package_io/input_parser.star

```diff
@@ -123,6 +125,13 @@ def parse_input(input_args):
 			if default_snooper_enabled:
 				participant["snooper_enabled"] = default_snooper_enabled
 
+		validator_count = participant["validator_count"]
+		if validator_count == None:
+			default_validator_count = result["network_params"]["num_validator_keys_per_node"]
+			participant["validator_count"] = default_validator_count
```

Comment:
if we set the default to 64, then this would be an non issue

Reply:
we also wouldn't be able to quickly override the default to non 64.